code/modules/client/preferences.dm

```diff
@@ -97,10 +97,18 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	if(istype(C))
 		if(!is_guest_key(C.key))
 			load_path(C.ckey)
+			if(!fexists(path))
+				try_savefile_type_migration()
 			unlock_content = !!C.IsByondMember()
 			if(unlock_content)
 				max_save_slots = 8
-
+	else
+		#ifdef UNIT_TESTS
+		load_path("unit_tests_dummy_ckey")
```

Comment:
I can't think of a better way to do this without adding more to mock_client (which you can do but is more work), but at least put this in tmp/ for people who run tests locally (like me)